test/argc_bug/rev-test-basim.py

```diff
@@ -22,17 +22,17 @@
 # Define the simulation components
 comp_cpu = sst.Component("cpu", "revcpu.RevCPU")
 comp_cpu.addParams({
-        "verbose" : 5,                                # Verbosity
-        "numCores" : 1,                               # Number of cores
-        "clock" : "1.0GHz",                           # Clock
-        "memSize" : 1024*1024*1024,                   # Memory size in bytes
-        "machine" : "[0:RV64IMFDC]",                  # Core:Config
-        "startAddr" : "[0:0x00000000]",               # Starting address for core 0
-        "memCost" : "[0:1:10]",                       # Memory loads required 1-10 cycles
-        "program" : os.getenv("REV_EXE", "dram_tests.exe"),  # Target executable
-        "args"    : "64 4 2 3",                       # Target command line arguments
-        "trcStartCycle" : "1",                        # Trace all instructions
-        "splash" : 1                                  # Display the splash message
```

Comment:
`flake8` considers it bad style to put spaces before `:` colons. The `key: value` should have no spaces between the key and the colon, and one or more spaces between the colon and the value.